PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1324,6 +1324,7 @@ Find in all files except exe, obj &amp;&amp; log:
             <find-status-replaceinopenedfiles-nb-replaced value="Replace in Opened Files: $INT_REPLACE$ occurrences were replaced"/>
             <find-status-mark-re-malformed value="Mark: The regular expression to search is malformed"/>
             <find-status-invalid-re value="Find: Invalid regular expression"/>
+            <find-status-search-failed value="Find: Search failed"/>
```

Comment:
Reword the message.

Reply:
The message should help the user understand and solve the issue.  _Why_ did the search fail?  What should they do about it?

Reply:
The message _does_ help the user understand what went wrong:

![image](https://user-images.githubusercontent.com/30118311/111210930-38dbed00-85a4-11eb-8b3a-b12217d065d4.png)

As far as what to do about it, well, for the pictured case that may require some good understanding of regex (on how to recraft the regex so that it might be simpler as to not cause the engine any troubles, if possible) but I don't think that is something Notepad++ is going to be able to tell you how to solve.

In other cases, it does pinpoint an invalid regex problem, example:

![image](https://user-images.githubusercontent.com/30118311/111211300-9ec87480-85a4-11eb-891b-783637fbf5dd.png)

Reply:
@sasumner Brilliant!

Reply:
@donho LOL, OK...but I didn't do anything but make some nice screenshots! :-)